pytket/docs/build-docs.sh

```diff
@@ -8,9 +8,9 @@ PYTKET_VERSION="$(pip show pytket| grep Version | awk '{print $2}')"
 PACKAGE="pytket $PYTKET_VERSION"
 
 # Build the docs setting the html title to show the correct pytket version.
-sphinx-build -W -b html . build -D html_title="$PACKAGE API documentation"
+sphinx-build -W -b html . build -D html_title="$PACKAGE API documentation" || exit 1
```

Comment:
Good spot with the `|| exit 1`

You can also use exit flags `+/-e` like I did in this PR -> https://github.com/CQCL/pytket-qiskit/pull/459/files#diff-24d28dd390892366ca6afd4b10fd1c0bcf44bffdd7735881e341c44d051d4fcdR42

However the effect should be the same I think so happy leaving this as is.